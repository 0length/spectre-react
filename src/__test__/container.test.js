import React from 'react'
import { shallow, render } from 'enzyme'
import renderer from 'react-test-renderer'

import { Container } from '../'

describe('Container', () => {
  it('Should render default .container markup', () => {
    const wrapper = shallow(<Container />)

    expect(wrapper.html()).toBe('<div class="container"></div>')
  })

  describe('Snapshots', () => {
    it('Should exist', () => {
      const wrapper = shallow(<Container />)

      expect(wrapper).toMatchSnapshot()
    })
  })
})
