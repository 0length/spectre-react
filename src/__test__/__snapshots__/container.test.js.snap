// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Snapshots Should exist 1`] = `
<div
  className="container"
/>
`;
